server/game/cards/12-SoW/HighHouseOfLight.js

```diff
@@ -0,0 +1,43 @@
+const StrongholdCard = require('../../strongholdcard.js');
+const { CardTypes, Players } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class HighHouseOfLight extends StrongholdCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Give a character with attachments bonus skill',
+            cost: AbilityDsl.costs.bowSelf(),
+            condition: () => this.game.isDuringConflict(),
+            target: {
+                cardType: CardTypes.Character,
+                controller: Players.Self,
+                cardCondition: card => card.isParticipating() && card.hasTrait('monk'),
+                gameAction: AbilityDsl.actions.cardLastingEffect(() => ({
+                    effect: AbilityDsl.effects.cardCannot({
+                        cannot: 'target',
+                        restricts: 'opponentsEvents'
+                    })
+                }))
+            },
+            then: context => ({
+                condition: context => this.game.currentConflict.getNumberOfCardsPlayed(context.player) >= 5,
+                gameAction: AbilityDsl.actions.selectRing({
+                    activePromptTitle: 'Choose a ring to take a fate from',
+                    ringCondition: ring => ring.fate >= 1,
```

Comment:
This shouldn't be necessary

Reply:
tested, it's required.  Otherwise can select a ring with no fate

Reply:
Pretty sure that's a bug in SelectRingAction - issue raised here #3850